unit-test

```diff
@@ -13,10 +13,21 @@ set -x
 build=$(mktemp -d -p $here)
 cd $build
 
-../cmake-init.sh
-ninja aktualizr-lite t_lite-helpers libt_lite-mock.so uptane-generator make_ostree_sysroot
-
-ninja aktualizr_clang_tidy-src-helpers.cc  aktualizr_clang_tidy-src-main.cc
+../cmake-init.sh -DCLANG_TIDY=ON
+ninja aklite-tests
 
 ctest -V -R test_lite-helpers
 ctest -V -R test_aktualizr-lite
+
+# test building the sota tools
+rm -rf ./*
+../cmake-init.sh -DBUILD_DOCKERAPP=OFF -DBUILD_SOTA_TOOLS=ON
+ninja all
```

Comment:
does this mean we'll be doing clang tidy against all of aktualizr also? or just the stuff under `src` and `tests`?

Reply:
Just stuff under src as it was before, I can make it for both src and tests, but currently, clang-tody is not happy about the stuff under tests, which implies that I have to make it happy before apply it.

Reply:
oops - sorry about that. don't worry about it on tests for now.